packages/rule-typescript-config-is-valid/src/rule.ts

```diff
@@ -0,0 +1,135 @@
+/**
+ * @fileoverview `typescript-config-is-valid` warns again providing an invalid typescript configuration file `tsconfig.json`.
+ */
+import * as ajv from 'ajv';
+import * as _ from 'lodash';
+
+import { Category } from 'sonarwhal/dist/src/lib/enums/category';
+import { RuleScope } from 'sonarwhal/dist/src/lib/enums/rulescope';
+import { RuleContext } from 'sonarwhal/dist/src/lib/rule-context';
+import { IRule, RuleMetadata } from 'sonarwhal/dist/src/lib/types';
+import { TypeScriptConfigInvalid, TypeScriptConfigInvalidSchema } from '@sonarwhal/parser-typescript-config/dist/src/types';
+import { debug as d } from 'sonarwhal/dist/src/lib/utils/debug';
+
+const debug: debug.IDebugger = d(__filename);
+
+/*
+ * ------------------------------------------------------------------------------
+ * Public
+ * ------------------------------------------------------------------------------
+ */
+
+export default class TypeScriptConfigIsValid implements IRule {
+    public static readonly meta: RuleMetadata = {
+        docs: {
+            category: Category.interoperability,
+            description: `\`typescript-config-is-valid\` warns again providing an invalid typescript configuration file \`tsconfig.json\``
```

Comment:
Maybe use ' instead of ` for the string so you don't have to escape the character?